packages/passport-interface/src/index.ts

```diff
@@ -1,4 +1,15 @@
-export * from "../test/MockFeedApi";
```

Comment:
No idea why this is so confusingly presented - the above line is the only thing removed.

Reply:
it probably auto-sorted the exports